sdk/core/Azure.Core.Expressions.DataFactory/src/DataFactoryLinkedServiceReference.Serialization.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace Azure.Core.Expressions.DataFactory
 {
+    [JsonConverter(typeof(DataFactoryLinkedServiceReferenceConverter))]
```

Comment:
This is required to serialize the type from the mgmt library. It was already added to the Core DF types.